### DISCUSSION TITLE
Asking for importing realm on a quay.io/keycloak/keycloak:15.1.1 with kubernetes

### DISCUSSION THREAD

**Discussion by aruffie** [keycloak/keycloak#28195] [Q&A]
Dear all,

I trying to deploy a Kubernetes deployment with quay.io/keycloak/keycloak:15.1.1 docker image.
But I'm facing to a problem that, after keycloak launching, I need to:

- import a realm
- create 2 users
- switch the authentication "direct grant flow"  to "direct grand"

I saw that a KEYCLOAK_IMPORT variable existing, but I'm not sure about the availability in 15.1.1 ?

someone can help me ?

Bellow my kubernetes config:

![image](https://github.com/keycloak/keycloak/assets/80361700/2b1488c6-1f81-4aaf-9c06-988974afff13)
![image](https://github.com/keycloak/keycloak/assets/80361700/c65ee874-4d2a-40ac-bce7-a105fc7cf878)

Thank a lot and best regards

Adrien

**Comment by codespearhead** [comment]
My suggestion: don't.

[Keycloak v15.x was superseded over 2 years ago](https://github.com/keycloak/keycloak/tags?after=16.1.0).

[Keycloak 22.x](https://access.redhat.com/products/red-hat-build-of-keycloak/) is the first Keycloak version that has an equivalent [commercial offering](https://access.redhat.com/announcements/7045552), ie. that's the de factor first LTS release of the project.

As for the community edition, [only the latest major release will get updates](https://github.com/keycloak/keycloak/discussions/10449#discussioncomment-2249604).

You can see how to deploy the latest Keycloak to Kubernetes [here](https://www.keycloak.org/getting-started/getting-started-kube).
